{
    "config": {
        "abort": {
            "already_configured": "El dispositivo ya est\u00e1 configurado",
            "heater_not_available": "Calentador no disponible. Intente restablecer el calentador pulsando + y OK durante algunos segundos.",
            "heater_not_found": "No se encuentra el calefactor. Intente acercar el calefactor al ordenador del Asistente de Hogar.",
            "invalid_auth": "Autenticaci\u00f3n no v\u00e1lida"
        },
        "error": {
            "cannot_connect": "No se pudo conectar",
            "invalid_auth": "Autenticaci\u00f3n no v\u00e1lida"
        },
        "step": {
            "cloud": {
                "data": {
                    "account_id": "ID de la cuenta",
                    "password": "Contrase\u00f1a"
                }
            },
            "local": {
                "data": {
                    "wifi_pswd": "Contrase\u00f1a Wi-Fi",
                    "wifi_ssid": "SSID Wi-Fi"
                }
            },
            "user": {
                "data": {
                    "account_id": "ID de la cuenta",
                    "host": "Host",
                    "password": "Contrase\u00f1a"
                }
            }
        }
    }
}